gnovm/pkg/gnolang/machine.go

```diff
@@ -1895,6 +1903,12 @@ func (m *Machine) PushFrameCall(cx *CallExpr, fv *FuncValue, recv TypedValue, is
 	// NOTE: fr cannot be mutated from hereon, as it is a value.
 	// If it must be mutated after append, use m.LastFrame() instead.
 	m.Frames = append(m.Frames, fr)
+	if len(m.Frames) > m.MaxFrames {
+		m.MaxFrames = len(m.Frames)
+		if m.GasMeter != nil {
+			m.GasMeter.ConsumeGas(GasCostFrame, "frame pushed")
```

Comment:
shouldn't we measure the size of the frame being pushed instead of having a fixed cost?
or maybe all frames have the same size?